### DISCUSSION TITLE
Continuous time models

### DISCUSSION THREAD

**Discussion by DEHewitt** [TheoMichelot/hmmTMB#28] [General]
Hi @TheoMichelot,

First of all, I am stoked you've developed such an easy to use package that enables the use of splines in a HMM context. 

I note in the pre-print that you mention continuous time models are a priority for future releases. Any estimate of the timeline? I'd _love_ to see this option available!

Cheers,
Dan

**Comment by TheoMichelot** [comment]
Hi Dan,

I have taken a number of notes over the last two years about what changes need to be made to implement continuous-time HMMs, but I haven't had time to do it yet as it's a fairly substantial job. Unfortunately I can't commit to a specific timeline. But I do tend to have more time for this kind of thing between April - August! And of course, anyone is welcome to submit a pull request in the meantime.
